Tools/TestWebKitAPI/Configurations/TestWebKitAPI.xcconfig

```diff
@@ -122,6 +122,7 @@ CODE_SIGNING_ALLOWED_YES = YES;
 CODE_SIGN_IDENTITY[sdk=macosx*] = $(CODE_SIGN_IDENTITY_$(CONFIGURATION));
 CODE_SIGN_IDENTITY_Debug = $(CODE_SIGN_IDENTITY_$(USE_INTERNAL_SDK));
 CODE_SIGN_IDENTITY_Release = $(CODE_SIGN_IDENTITY_$(USE_INTERNAL_SDK));
+CODE_SIGN_IDENTITY_ = -;
```

Comment:
Is the issue here that we are building these tools as neither Debug nor Release? I wonder if that's what needs to be fixed.

Reply:
I don't think so. Left a comment on the bug with my understanding of what's going on (tl;dr EWS has never used the entitlements we claim it does, and turning on ad-hoc signing reveals that those entitlements require a proper signing identity): https://bugs.webkit.org/show_bug.cgi?id=242730#c11

Reply:
I haven't read the comment yet. However, looking just at this one line in the PR, we wouldn't need it unless CONFIGURATION was empty in practice, correct?

Reply:
Oh I see what you mean. This is used when USE_INTERNAL_SDK is empty.